ui/src/app/views/homepage/component.css

```diff
@@ -1,251 +1,221 @@
-.front-page {
-    margin-left: -1rem;
-    margin-right: -.6rem;
-    text-align: center;
-    position: relative;
-}
-
 .background-image {
-    background-image: url('/assets/images/homepage.png');
+    background-image: url('/assets/images/AoU-HP-background.jpg');
     background-repeat: no-repeat;
-    z-index: -1;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
     background-size: cover;
-    /*height: calc(100vh - 3.3rem);*/
-    height: 41rem;
-}
-
-.rectangle {
-    opacity: .47;
-    background-color: #000000;
-    box-shadow: 0 1px 4px 0 rgba(0,0,0,0.28);
-    position: absolute;
-    margin-top: 5rem;
-    left: 50%;
-    transform: translate(-50%, 0);
-    width: 80%;
-    height: 11rem;
-    z-index: 1;
-}
-
-.invisible_rectangle {
-    margin-top: 5rem;
-    width: 80%;
-    z-index: 2;
-    height: 11rem;
-    position: absolute;
-    left: 50%;
-    transform: translate(-50%, 0);
-}
-
-.tip-title {
-    color: #FFFFFF;
-    padding-top: 1rem;
-    padding-left: 2rem;
-    font-size: 1.8rem;
-    font-weight: 300;
-    line-height: 2rem;
-    position: absolute;
+    height:100%
 }
 
-.tip-subtitle {
-    padding-top: 3.7rem;
-    padding-left: 2.1rem;
-    font-size: .9rem;
-    line-height: 1rem;
+.card {
+    border-radius: 5px;
 }
 
-.tip-description {
-    padding-top: 5rem;
-    font-size: .7rem;
-    line-height: .9rem;
-    font-weight: lighter;
-    text-align: left;
-    width: 55%;
+.double {
+    height: 332px;
+    width: 548px;
+    background-color: rgba(255,255,255,0.15);
+    box-shadow: 0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12);
 }
 
-.learn-more-button {
-    display: flex;
-    align-content: center;
-    justify-content: center;
-    flex-direction: column;
-    height: 1.5rem;
-    width: 6.5rem;
-    -webkit-border-radius: .3125rem;
-    -moz-border-radius: .3125rem;
-    border-radius: .3125rem;
-    background-color: #60AEE0;
-    position: absolute;
-    color: #FFFFFF;
-    margin-top: 9rem;
-    margin-left: 2rem;
-    font-weight: 300;
-    font-size: .6rem;
-    cursor: pointer;
+.left {
+    margin-left: 301px;
 }
 
-.learn-more-button:hover {
-    background-color: #80C3ED;
+.right {
+    margin-left: 23px;
 }
-.profile-box {
-    height: 11rem;
-    background-color: #FFFFFF;
-    box-shadow: 0 1px 4px 0 rgba(0,0,0,0.28);
-    float: right;
-    position: relative;
-    width: 15rem;
-    z-index: 1;
-    text-align: left;
-}
-
-.profile-title {
-    width: 100%;
-    color: rgba(0, 0, 0, 0.87);
-    line-height: 1rem;
-    font-size: .75rem;
-    font-weight: 300;
-    padding-top: 1.2rem;
-    padding-left: 1.2rem;
-    padding-right: .2rem;
+
+.single {
+    height: 349px;
+    width: 1118px;
+    border-radius: 5px;
+    background-color: rgba(255,255,255,0.15);
+    box-shadow: 0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12);
```

Comment:
This has a lot similar to `.double`. Can we abstract that to the card class? Or just do a shared class and then only have width different?